Command/LoadYamlFixturesCommand.php

```diff
@@ -66,8 +65,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         if ($input->isInteractive() && !$input->getOption('append')) {
-            /* @var DialogHelper $dialog */
-            $dialog = $this->getHelperSet()->get('dialog');
+            $helperName = class_exists('Symfony\Component\Console\Helper\QuestionHelper') ? 'question' : 'dialog';
```

Comment:
@tomas-pecserke I am sorry to spoil you with something that can't be really used on Symfony 2.3. I've thought I am on 2.3 branch and I was happy to see it there, but I see it's master now and it's not available on 2.3